CHANGELOG.md

```diff
@@ -13,17 +13,21 @@ Please use [pce-fw-builder](https://github.com/pcengines/pce-fw-builder)
 
 ## [Unreleased]
 
+## [v4.12.0.7] - 2020-11-27
+### Changed
+- rebased with official coreboot repository commit 957a363
+
```

Comment:
@Sakoram I have merged a PR to develop: https://github.com/pcengines/coreboot/pull/447

It fixes the vboot issues with reset in early boot flow.